ecs-init/volumes/ecs_volume_plugin_test.go

```diff
@@ -189,11 +189,18 @@ func TestVolumeCreateFailure(t *testing.T) {
 	createMountPath = func(path string) error {
 		return nil
 	}
+
+	var removeInvoked bool
+	removeMountPath = func(path string) error {
```

Comment:
this should be re-assigned to the original method in defer block.

Reply:
good catch. fixed